### ISSUE TITLE
Combat support: nice to haves

### ISSUE THREAD

**Issue by Alorel** [Alorel/melvor-action-workflows#37]
Blocked by #36

- Triggers:
   - HP:
      - [ ] `>=` / `<=` a number or %
      - [ ] `<=` incoming damage
         - Might conflict with autorun scripts
         - Take own debuff & enemy attack time into account
   - Enemy has/doesn't have buff/debuff
   - Player has/doesn't have debuff
